file-formats/ilmb/type/zif_aff_ilmb_v1.intf.abap

```diff
@@ -0,0 +1,108 @@
+INTERFACE zif_aff_ilmb_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Condition Fields Direct</p>
+    "! Condition fields direct
+    BEGIN OF ty_cond_fields_direct,
+      "! <p class="shorttext">Condition Field</p>
+      "! Condition field
+      condition_field TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Source Table</p>
+      "! Source table
+      source_table    TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Source Field</p>
+      "! Source field
+      source_field    TYPE zif_aff_types_v1=>ty_object_name_30,
+    END OF ty_cond_fields_direct,
+    "! <p class="shorttext">Condition Fields Direct</p>
+    "!  Condition fields direct
+    ty_cond_fields_directs TYPE SORTED TABLE OF ty_cond_fields_direct WITH UNIQUE KEY source_table source_field.
+
+  TYPES:
+    "! <p class="shorttext">Available Condition Fields</p>
+    "! Available condition fields
+    BEGIN OF ty_available_condition_field,
+      "! <p class="shorttext">Condition Field</p>
+      "! Condition field
+      condition_field             TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Data Element</p>
+      "! Data element
+      data_element                TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Data Element Description</p>
+      "! Data element description
+      data_element_description    TYPE zif_aff_types_v1=>ty_description_60,
+      "! <p class="shorttext">Field Name Description</p>
+      "! Field name description
+      condition_field_description TYPE zif_aff_types_v1=>ty_description_60,
+      "! <p class="shorttext">No Intervals</p>
+      "! No intervals
+      no_intervals                TYPE abap_bool,
+    END OF ty_available_condition_field,
+    "! <p class="shorttext">Available Condition Fields</p>
+    "! Available condition fields
+    ty_available_condition_fields TYPE SORTED TABLE OF ty_available_condition_field WITH UNIQUE KEY condition_field.
+
+  TYPES:
+    "! <p class="shorttext">Destruction Object</p>
+    "! Destruction object
+    BEGIN OF ty_destruction_object,
+      "! <p class="shorttext">Destruction Object</p>
+      "! Destruction object
+      destruction_object  TYPE c LENGTH 30,
```

Comment:
First, I had in mind to rename the structure to "General Information". But I think I prefer to keep it as "Destruction Object" and just rename the field to `name`. 

```suggestion
      name  TYPE zif_aff_types_v1=>ty_object_name_30,
```

Reply:
Adapted as suggested.